### PR TITLE
BAEL-2981: Added code samples for Handlebars.java

### PR DESCRIPTION
**Pull request by isaolmez**

### CODE REVIEW

File: libraries-2/pom.xml [hunk 1]
```diff
@@ -81,13 +81,17 @@
...
+            <groupId>edu.uci.ics</groupId>
+            <artifactId>crawler4j</artifactId>
+            <version>${crawler4j.version}</version>
+        </dependency>
+        <!-- HandleBars -->
+        <dependency>
+            <groupId>com.github.jknack</groupId>
+            <artifactId>handlebars</artifactId>
+            <version>4.1.2</version>
+        </dependency>
```
Comment (rpvilao) [inline libraries-2/pom.xml]: format

File: libraries-2/src/test/java/com/baeldung/handlebars/BasicUsageUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Template;
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.TemplateLoader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class BasicUsageUnitTest {
```
Comment (rpvilao) [inline libraries-2/src/test/java/com/baeldung/handlebars/BasicUsageUnitTest.java]: header describing what's being tested and author as well

Comment (isaolmez) [reply libraries-2/src/test/java/com/baeldung/handlebars/BasicUsageUnitTest.java]: Fixed.

File: libraries-2/pom.xml (hunk 1 above)
Comment (isaolmez) [reply libraries-2/pom.xml]: Fixed.

**Pull request merged**